src/Abstractions/NexusMods.Abstractions.FileStore/Downloads/DownloadAnalysis.cs

```diff
@@ -101,6 +102,21 @@ public ModFileTreeNode GetFileTree(IFileStore? fs = null)
         {
             return TreeCreator.Create(Contents, fs);
         }
+        
+        /// <summary>
+        /// The timestamp of the creation of this <see cref="DownloadAnalysis"/> entity
+        /// Download start time or first manual installation time
+        /// </summary>
+        public DateTime CreatedAt
+        {
+            get
+            {
+                // Get the lowest transaction id, then get the timestamp of that transaction
+                var t = this.Select(d => d.T).Min();
+                var txEntity = Db.Get<Entity>(EntityId.From(t.Value));
+                return BuiltInAttributes.TxTimestanp.Get(txEntity);
+            }
+        }
```

Comment:
Should be a method `GetCreatedAt` instead of a property.

Reply:
I copied what was already on the Mod entity:
https://github.com/Nexus-Mods/NexusMods.App/blob/db8503381d4e3b1946cb8df8a09db77b6a396ba1/src/Abstractions/NexusMods.Abstractions.Loadouts/Mods/Mod.cs#L165

Should I change both or leave as is?

Reply:
Change the other as well.